resources/cmake/ConnextDdsAddExample.cmake

```diff
@@ -364,7 +368,7 @@ endfunction()
 
 function(connextdds_copy_qos_profile)
     set(optional_args)
-    set(single_value_args TARGET_PREFIX DEPENDANT_TARGET)
+    set(single_value_args TARGET_PREFIX DEPENDANT_TARGET FILENAME)
```

Comment:
Please, could you add the `FILENAME` parameter to the documentation of this CMake file?

Reply:
Yes will do.

Reply:
WIll do.